arbitrum-docs/run-arbitrum-node/06-troubleshooting.mdx

```diff
@@ -144,98 +144,260 @@ If you're running into unexpected outputs or errors, the following checklist may
+          </td>
+          <td>
+            This often happens when Cloudflare attempts to block botnets and other malicious actors
+            but accidentally blocks node runners.
+          </td>
+        </tr>
+        <tr className="scenario">
+          <td>
+            You see <code>latest assertion not yet in our node</code>
+          </td>
+          <td>
+            This usually because your node hasn’t synced to the latest state, it’s a normal
+            behavior.
+          </td>
+        </tr>
+        <tr className="scenario">
+          <td>
+            You see <code>"Post "xxx_url": context deadline exceeded"</code>
+          </td>
+          <td>
+            Please check your parent chain endpoint because there is something going wrong on that
+            endpoint; you can check it.
+          </td>
+        </tr>
+        <tr className="scenario">
+          <td>
+            You see <code>Resuming state snapshot generation</code>
+          </td>
+          <td>
+            This is a normal behavior while the node is catching up to the tip of the chain; once a
+            node has been fully synced, "Resuming state snapshot generation" shouldn't be logged
+            unless it falls behind again.
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </TabItem>
+  <TabItem value="warn">
+    <table className="small-table" id="troubleshooting-table-warn">
+      <tbody>
+        <tr>
+          <th style={{ minWidth: 180 + 'px' }}>Scenario</th>
+          <th>Solution</th>
+        </tr>
+        <tr className="scenario">
+          <td>
+            You see{' '}
+            <code>
+              `error reading inbox` err="sequencer batches out of order; after batch A got batch B”
+            </code>
+          </td>
+          <td>
+            This is because you get two discontinuous batches; this might be because of your parent
+            chain endpoint issues. You can change to another endpoint and set nitro `
+            --init.reorg-to-batch A`
+          </td>
+        </tr>
+        <tr className="scenario">
+          <td>
+            You see{' '}
+            <code>
+              `error reading inbox` err="failed to get previous message for pos `x`: leveldb: not
+              found”
+            </code>
+          </td>
+          <td>
+            This is because your node db got crashed and lost some message, you can try set
+            `--init.reorg-to-message-batch x-1`
+          </td>
+        </tr>
+        <tr className="scenario">
+          <td>
+            You see{' '}
+            <code>`Failed to load snapshot` err="head doesn't match snapshot: have a, want b”</code>
+          </td>
+          <td>
+            This is usually because an ungraceful shutdown causes a corrupted database; try
+            restarting the node without a prune flag, and after your node goes back to normal, then
+            graceful shutdown it and restart to prune it.
```

Comment:
```suggestion
            This is usually because an ungraceful shutdown caused a corrupted database; try
            restarting the node without a prune flag, and after your node goes back to normal, then
            graceful shut it down and restart to prune it.
```